README.md

```diff
@@ -73,12 +58,28 @@ For example:
         ```
         db.inventory('2MASS J01415823-4633574', pretty_print=True)
         ```
+5. The database can also be modified using helper scripts found in [`simple/utils`](simple/utils) and in the `astrodb_utils` package. Previously used scripts to modify and/or update the database are stored in the [`scripts/`](scripts) directory and can be used for inspiration.
+
+## Contributor Instructions
+If you've made changes to the SIMPLE Archive that you would like to contribute to the public verion, here's how to make a contribution.
+
+1. If you've made changes using Python or the DB Browser, write out the new/modified JSON files:
```

Comment:
Suggested rewrite:
`If you've made changes to the database contents, write out the new/modified JSON files with:`
(no need to specify python/DB browser)